### PR TITLE
Add boilerplate code

### LINKED ISSUE #3: Initialize new component with standard MeshKit functions
---
**[Contributor Guide](https://docs.meshery.io/project/contributing)**

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
**Description**

This PR fixes #3

- [ ] Initialised new component with MeshKit functions
- [x] Update build scripts (Makefile)
- [x] Create Dockerfile
- [x] Create build and publish workflow

**Notes for Reviewers**

@Revolyssup , @utkarsh-pro Do we need to define CRDs in order to create components? The url for the manifest file is not created now. Could you guide on how to proceed from here?

- [x] Yes, I signed my commits.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,163 @@
+module github.com/meshery/meshery-perf
```
Comment (Revolyssup) [inline go.mod:1]: We would not need this file. As far as I know, meshery perf will not register any definitions and schemas. So for that reason, this file is not needed. We are not registering any components here

File: main.go [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+	// Initialize Handler intance
+	handler := perf.New(cfg, log, kubeconfigHandler)
+	handler = adapter.AddLogger(log, handler)
+
+	service.Handler = handler
+	service.Channel = make(chan interface{}, 10)
+	service.StartedAt = time.Now()
+	service.Version = version
+	service.GitSHA = gitsha
+	go registerCapabilities(service.Port, log)            //Registering static capabilities
```
Comment (Revolyssup) [inline main.go:98]: I think these two go routines can be removed.

File: perf/oam/register.go [hunk 3]
```diff
@@ -0,0 +1,127 @@
+package oam
```
Comment (Revolyssup) [inline perf/oam/register.go:1]: This file can be removed as well.

**Review by Revolyssup — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
@navendu-pottekkat ping me and I'll get you going.

**Comment by leecalcote** [comment]
@navendu-pottekkat @hershd23 @gyohuangxin I have added/updated workflows, added dockerfile and makefile.

**Comment by leecalcote** [comment]
Merging to unblock and enable progress...

**Pull request merged**